README.md

```diff
@@ -1,41 +1,61 @@
 # xbox kernel test suite
-Xbox kernel APIs tester written using the opensoure nxdk
+[![License: GPL v2](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](https://github.com/Cxbx-Reloaded/xbox_kernel_test_suite/blob/master/LICENSE)
+[![GitHub CI](https://github.com/Cxbx-Reloaded/xbox_kernel_test_suite/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/Cxbx-Reloaded/xbox_kernel_test_suite/actions/workflows/ci.yml?query=branch%3Amaster)
 
-This is a tool for testing xbox kernel apis, in particular CxBx-Reloaded kernel implementation.
+Xbox kernel APIs tester written using the open-source nxdk.
 
-# HOW TO BUILD:
-All you need is nxdk. You can get it here: https://github.com/xqemu/nxdk
+This is a tool for testing the xbox kernel APIs on the hardware to verify tests are passing. Plus we have a dedicated wiki page for [homebrew kernels](https://github.com/Cxbx-Reloaded/xbox_kernel_test_suite/wiki/List-of-Homebrew-Kernels-Tested) have used this tool to test their kernel implementation.
 
-Here is a setup guide: https://github.com/xqemu/nxdk/wiki/Getting-Started
+## HOW TO BUILD:
+All you need is nxdk. You can get it here: https://github.com/XboxDev/nxdk
 
-# CONFIGURATION FILE:
-The configuration file should be called _**"config.txt"**_ and should be placed in the same directory of the xbe.
+Here is a setup guide: https://github.com/XboxDev/nxdk/wiki/Getting-Started
 
-This is an example of config.txt:
+## CONFIGURATION FILE:
+> [!NOTE]
+> The configuration file should be called _**"config.txt"**_ and should be placed in the same directory of the xbe.
 
-```
-seed=5
+The following list of options can be used inside config.txt file:
+- `seed` = `<hexadecimal (support up to unsigned int max)>`
+- `tests` = `<hexadecimal (support up to 17A)>[,...]`
+- `tests-exclude` = `<hexadecimal (support up to 17A)>[,...]`
+- `disable-video` = `<boolean>`[^1]
 
-tests=1,25,3,F
-```
+[^1]: boolean value can be 1 or 0
```

Comment:
This is not true. The code for the disable-video option is using a `if` statement to check if it should output to the video. This means that any value other than zero will output, not just one like this line claims.

Reply:
Technically, 1-9A-F characters will be read as "true" which then will make disable video due to inverter. If no input or zero will be treat as enable video by default.

But this is more or less a guideline should for readers to follow properly.